phinx.php

```diff
@@ -12,14 +13,14 @@
   ],
   'environments' => [
     'default_migration_table' => 'phinxlog',
-    'default_environment' => 'development',
-    'production' => [
-      'dsn' => "pgsql://${_ENV['POSTGRES_USER']}:${_ENV['POSTGRES_PASSWORD']}@${_ENV['POSTGRES_HOST']}/${_ENV['POSTGRES_DB']}"
+    'default_environment' => 'dev',
+    'prod' => [
```

Comment:
@okiedork should this be kept as "production"?

Reply:
`prod`.